stdlib/buffer.mli

```diff
@@ -177,3 +177,34 @@ val add_seq : t -> char Seq.t -> unit
 val of_seq : char Seq.t -> t
 (** Create a buffer from the generator
     @since 4.07 *)
+
+(** {6 Binary storage of numbers} *)
+
+val add_int16_ne : t -> int -> unit
```

Comment:
I think this should be:
```
(** [add_int16_ne b i] appends a binary native endian signed 16-bit integer [i] to [b]. @since NEXT_RELEASE *)
```
So the pattern is:
```
(** [add_intXX_EE b i] appends a binary EE endian signed XX-bit integer [i] to [b]. @since NEXT_RELEASE *)
```

Reply:
I think "EE-endian" would be clearer (and probably more grammatically correct, since it is being used as an adjective).

Reply:
- Don't you want to mention the behavior when [i] does not fit in 16-bit?

- I don't think that mentioning signedness for add/set functions make sense (the functions only truncates the binary representation); signed vs unsigned makes sense when reading, to control whether the result is sign-extended or not.

- Is native-endian a common terminology?  Shouldn't the documentation stresses that the behavior is machine-dependent?